### PR TITLE
Issue #SB-23540: Config changes for sourcing summary job

### PR DESCRIPTION
**Pull request by ishawakankar**
<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/SB-23540" title="SB-23540" target="_blank">EpicSB-23540</a>  Reports to be generated from Hawkeye infra using Druid
  </td></table>
  <br />

### CODE REVIEW

File: ansible/roles/data-products-deploy/files/sourcing-ingestion-spec.json [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+                  "expr": "status"
+                },
+                {
+                  "type": "root",
+                  "name": "rootorg_id",
+                  "expr": "rootorg_id"
+                },
+                {
+                  "type": "root",
+                  "name": "userId",
```
Comment (manjudr) [inline ansible/roles/data-products-deploy/files/sourcing-ingestion-spec.json:30]: Please follow the ingestion spec standard

File: ansible/roles/data-products-deploy/tasks/main.yml [hunk 2]
```diff
@@ -264,6 +264,11 @@
   tags:
     - ed-dataproducts
```
Comment (manjudr) [inline ansible/roles/data-products-deploy/tasks/main.yml:266]: Please add a cron job task.

File: ansible/roles/data-products-deploy/templates/model-dock-config.j2 [hunk 3]
```diff
@@ -9,13 +9,17 @@ config() {
 	sparkCassandraConnectionHost="{{groups['lp-cassandra'][0]}}"
 	sunbirdPlatformCassandraHost="{{ core_cassandra_host }}"
 	reportPostContainer="{{ reports_container }}"
+	druidRollupHost="{{ druid_rollup_cluster_ingestion_task_url }}"
```
Comment (manjudr) [inline ansible/roles/data-products-deploy/templates/model-dock-config.j2:12]: Hope this variable contains port as well.

**Review by manjudr — CHANGES_REQUESTED** [review]

File: ansible/roles/data-products-deploy/tasks/main.yml (hunk 2 above)
Comment (ishawakankar) [reply ansible/roles/data-products-deploy/tasks/main.yml:266]: Added

**Review by SanthoshVasabhaktula — APPROVED** [review]

**Pull request merged**